static/src/javascripts.flow.archive/bootstraps/enhanced/accessibility.js

```diff
@@ -11,7 +11,7 @@
  */
 
 // @flow
-import React, { Component, render } from 'preact-compat';
```

Comment:
did you mean to change the javascripts.flow.archive/ files?
(I don't know why we have this archive folder)

Reply:
They're a reference for the types if someone needs to type a file with Typescript (doesn't really matter whether the files are changed or not). 

Side note, maybe we could drop them into another repo as it makes search frustrating.

Reply:
![files to exclude](https://user-images.githubusercontent.com/76776/111334902-54013800-864a-11eb-8fd8-3244f6a66f20.png)


I just exclude `*flow.archive/`